packages/blog/gatsby/gatsby-browser.ts

```diff
@@ -1,7 +0,0 @@
-import "typeface-montserrat"
-import "typeface-merriweather"
-
-import "../src/index.css"
-
-// Highlighting for code blocks
-import "prismjs/themes/prism.css"
```

Comment:
temporarily removed styles (and crawler can parse blog posts now)

so the main reason is to reduce the styles from `<head>`